_posts/2019-03-22-cockpit-190.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Cockpit 190
+author: kkoukiou
+date: 2019-03-22
+tags: cockpit linux
+slug: cockpit-190
+category: release
+summary: Cockpit with new features in logs and machines
+comments: 'true'
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/). We
+release regularly.  Here are the release notes from version 190.
+
+### Logs: Allow filtering logs entries by services
```

Comment:
bad grammar (AFAIK), and a bit redundant. "Logs: Filter log entries by service".

Reply:
Done